index.html

```diff
@@ -2,12 +2,82 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    <section>
+
+        <div class="wrapper">
+            <header class="toolbar">
+                <nav>            
+                    <a href="#">
+                    <li class="item">Edahí Gutiérrez </li> </a>        
+                    <li id="item-right" class="item">&#9776;</li>                                
+                </nav>
+            </header>
+            
+        
+            <h1>Hey There. <br>
+                I’m Edahí</h1>
+        
+            <h2 id="intro">I’am a Software Developer</h2>   
```

Comment:
[OPTIONAL] As commented by the previous reviewer, this semantic can be improved. Please note that headers shoudn't be used because of font size (you can always edit it using css), they are helpful for accessibility and website structure. This means that, a webpage should have one `h1` that is the main header of all your content, and the rest of the page should be divided using `h2`, `h3` and so on. You can understand more about them [here](https://www.w3schools.com/html/html_headings.asp).

Reply:
changed to paragraphs